packages/truss/src/sections/tachyons/overflow.ts

```diff
@@ -5,22 +5,22 @@ import { newMethodsForProp } from "src/methods";
 export const overflow: CreateMethodsFn = () => {
   return [
     ...newMethodsForProp("overflow", {
-      overflowVisible: "visible",
-      overflowHidden: "hidden",
-      overflowScroll: "scroll",
-      overflowAuto: "auto",
+      ofv: "visible",
+      ofh: "hidden",
+      ofs: "scroll",
+      ofa: "auto",
```

Comment:
If I'm being a real stickler, then `overflow` -> `of` doesn't exactly follow the conventions, right? That'd expect "overFlow" -> `of`. I'd be okay with `ov`, `oh`, `os`, etc.

Reply:
Yep, updated